scriptmodules/ports/gmloader.sh

```diff
@@ -13,19 +13,25 @@
 rp_module_id="gmloader"
 rp_module_desc="GMLoader - play GameMaker Studio games for Android on non-Android operating systems"
 rp_module_help="ROM Extensions: .apk .APK\n\nIncludes free games Maldita Castilla and Spelunky Classic HD. Use launch scripts as template for additional games."
-rp_module_repo="git https://github.com/JohnnyonFlame/droidports.git master faf3970"
+rp_module_repo="git https://github.com/JohnnyonFlame/droidports.git"
```

Comment:
Needs to keep the branch name (master). See https://github.com/Exarkuniv/RetroPie-Extra/commit/9be91b8b46e1c55f6ad3c1df57ad9e01e3ba0454

Reply:
I'd like to challenge this as "master" is added when absent: https://github.com/RetroPie/RetroPie-Setup/blob/4a8597c39071e351ac52a7b5ed705c0554e70afb/scriptmodules/helpers.sh#L437

Reply:
It works fine in the `gitPullOrClone` function. However the `rp_module_repo` variable, is also used by the setup script to determine if updates are available; if the branch name is missing, it can't stat the version to compare with and won't tell you if an update is available or not. All you can do is uninstall, then reinstall:

![20230807_135148](https://github.com/Exarkuniv/RetroPie-Extra/assets/92833764/0e95d360-0513-4aae-9dad-f9f039c8acac)

However when the branch name is added back in, it can compare the installed version against the repo and tell you if an update is available, or you're running the latest version, and update on a single step of available:

![20230807_135220](https://github.com/Exarkuniv/RetroPie-Extra/assets/92833764/71b6b124-09d0-4a00-8d0a-5b2137177a7c)